clinica/pipelines/statistics_volume/statistics_volume_cli.py

```diff
@@ -0,0 +1,112 @@
+# coding: utf8
+
+# coding: utf-8
+
+__author__ = "Arnaud Marcoux"
+__copyright__ = "Copyright 2016-2019 The Aramis Lab Team"
+__credits__ = ["Arnaud Marcoux"]
+__license__ = "See LICENSE.txt file"
+__version__ = "0.3.0"
+__maintainer__ = "Arnaud Marcoux"
+__email__ = "arnaud.marcoux@icm-institute.org"
+__status__ = "Development"
+
+import clinica.engine as ce
+
+
+class StatisticsVolumeCLI(ce.CmdParser):
+
+    def define_name(self):
+        """Define the sub-command name to run this pipeline."""
+        self._name = 'statistics-volume'
+
+    def define_description(self):
+        """Define a description of this pipeline."""
+        self._description = ('Brief description:\n'
+                             'http://clinica.run/doc/Pipelines/Statistics_Volume/')
+
+    def define_options(self):
+        """Define the sub-command arguments."""
+        from clinica.engine.cmdparser import PIPELINE_CATEGORIES
+
+        clinica_comp = self._args.add_argument_group(PIPELINE_CATEGORIES['CLINICA_COMPULSORY'])
+
+        clinica_comp.add_argument("caps_directory",
+                                  help='Path to the CAPS directory.')
+
+        clinica_comp.add_argument("subject_visits_with_covariates_tsv",
+                                  help='TSV file containing a list of subjects with their sessions and all '
+                                       'the covariates and factors needed for the GLM.')
+
+        clinica_comp.add_argument("contrast",
+                                  help='Defines the contrast. Must be one of the column names form the TSV file.')
+
+        clinica_comp.add_argument("feature_type",
+                                  help='Define what type of file are grabbed for the analysis. Use "fdg" or "graymatter". Use your own if you want to use the --custom_file flag. ')
+
```

Comment:
In order to be homogeneous with `statistics-surface`, this needs to be a "Pipeline option".  The resulting help should looke like:
```python
        optional.add_argument("-ft", "--feature_type",
                              type=str, default='graymatter',
                              help='Type of volume-based feature: graymatter (from t1-volume pipeline) or '
                                   'fdg (from pet-volume pipeline) (default: --feature_type %(default)s).')
```
This will be very similar to `statsitics-surface`: 
```text
  -ft FEATURE_TYPE, --feature_type FEATURE_TYPE
                        Type of surface-based feature: cortical_thickness
                        (from t1-freesurfer pipeline)or pet_fdg_projection
                        (from pet-surface pipeline)(default: --feature_type
                        cortical_thickness).
```

Reply:
Agreed ! This is changed